tests/taskmodules/test_re_text_classification_with_indices.py

```diff
@@ -1058,26 +1065,74 @@ def test_encode_input_multiple_relations_for_same_arguments(caplog):
+            assert (
+                caplog.messages[0]
+                == "doc.id=multiple_relations_for_same_arguments: there are multiple relations with the "
+                "same arguments (('head', ('PER', 'A')), ('tail', ('PER', 'B'))): previous "
+                "label='per:founded_by' and current label='per:founder'. We only keep the first "
+                "occurring relation which has the label='per:founded_by'."
+            )
+            assert (
+                caplog.messages[1]
+                == "doc.id=multiple_relations_for_same_arguments: Relation annotation "
+                "`('per:founded_by', (('PER', 'A'), ('PER', 'B')))` is duplicated. We keep "
+                "only one of them."
+            )
+            assert (
+                caplog.messages[2] == "statistics:\n"
+                "|                        |   per:founded_by |   per:founder |   all_relations |\n"
+                "|:-----------------------|-----------------:|--------------:|----------------:|\n"
+                "| available              |                1 |             1 |               2 |\n"
+                "| skipped_same_arguments |                0 |             1 |               1 |\n"
+                "| used                   |                1 |             0 |               1 |\n"
+                "| used %                 |              100 |             0 |              50 |"
+            )
+            assert len(encodings) == 1
+            relation = encodings[0].metadata["candidate_annotation"]
+            assert str(relation.head) == "A"
+            assert str(relation.tail) == "B"
+            assert relation.label == "per:founded_by"
+        elif handle_relations_with_same_arguments == "keep_none":
+            assert (
+                caplog.messages[0]
+                == "doc.id=multiple_relations_for_same_arguments: there are multiple relations "
+                "with the same arguments (('head', ('PER', 'A')), ('tail', ('PER', 'B'))): "
+                "previous label='per:founded_by' and current label='per:founder'. Both relations "
+                "will be removed."
+            )
+            assert (
+                caplog.messages[1]
+                == "doc.id=multiple_relations_for_same_arguments: Relation annotation "
+                "`('per:founded_by', (('PER', 'A'), ('PER', 'B')))` is duplicated. "
+                "We keep only one of them."
+            )
+            assert (
+                caplog.messages[2] == "statistics:\n"
+                "|                        |   per:founded_by |   per:founder |   all_relations |\n"
+                "|:-----------------------|-----------------:|--------------:|----------------:|\n"
+                "| available              |                1 |             1 |               2 |\n"
+                "| skipped_same_arguments |                1 |             1 |               2 |"
+            )
+            assert len(encodings) == 0
```

Comment:
add a comment explaining this result (why zero entries, given input doc with xxx?)